### DISCUSSION TITLE
Java Swing/AWT. Nerd font family isn't found

### DISCUSSION THREAD

**Discussion by Lassebq** [ryanoasis/nerd-fonts#1256] [Q&A] [answered]
Java tries to find font family by the name "JetBrainsMono Nerd Font Medium", but it doesn't exits if I grep fc-list
The only font with a similar name that there is "JetBrainsMono NF Medium".
Is there any way I can "alias" a font to another font? The GTK font is set to "JetBrainsMono Nerd Font Bold"

![image](https://github.com/ryanoasis/nerd-fonts/assets/68742864/f1fda360-1ecc-465c-9589-915a9ed96335)

**Comment by Lassebq** [comment]
I tried using other non-nerd fonts and they work

**Comment by Finii** [comment, marked as answer]
There are the family names of all current release fonts:

```
| Family                                                 |
|--------------------------------------------------------|
| JetBrainsMono NF                                       |
| JetBrainsMono NF ExtraBold                             |
| JetBrainsMono NF ExtraLight                            |
| JetBrainsMono NF Light                                 |
| JetBrainsMono NF Medium                                |
| JetBrainsMono NF SemiBold                              |
| JetBrainsMono NF Thin                                  |
| JetBrainsMono NFM                                      |
| JetBrainsMono NFM ExtraBold                            |
| JetBrainsMono NFM ExtraLight                           |
| JetBrainsMono NFM Light                                |
| JetBrainsMono NFM Medium                               |
| JetBrainsMono NFM SemiBold                             |
| JetBrainsMono NFM Thin                                 |
| JetBrainsMono NFP                                      |
| JetBrainsMono NFP ExtraBold                            |
| JetBrainsMono NFP ExtraLight                           |
| JetBrainsMono NFP Light                                |
| JetBrainsMono NFP Medium                               |
| JetBrainsMono NFP SemiBold                             |
| JetBrainsMono NFP Thin                                 |
```

I'm not sure what `JetBrainsMono Nerd Font Medium` has to do with `JetBrainsMono Nerd Font Bold`; except both use the old name with `Nerd Font` instead of `NF`.

`JetBrainsMono Nerd Font Medium` is a self standing family.
`JetBrainsMono Nerd Font Bold` is the full name of the `Bold` subfamily of the family `JetBrainsMono Nerd Font`.

Why is "the GTK font" set to that value?

To answer the concrete question: Yes, you can add a rule to your fontconfig that aliases one font for another.
I believe there is even a commented out example typically in the fontconfig in `/etc`; some more examples are [here](https://wiki.archlinux.org/title/Font_configuration/Examples) or [here](https://jichu4n.com/posts/how-to-set-default-fonts-and-font-aliases-on-linux/).

**Reply by Lassebq** [reply]
Thank you, after manually changing the name to `JetBrainsMono NF Bold` java applications started using it. However, the application I use for switching themes (nwg-look) uses old name for some reason. And if I change it again it'll break java apps.
